tests/core/test_i64.in

```diff
@@ -0,0 +1,29 @@
+#include <stdio.h>
```

Comment:
This matches the `.in`/`.out` file types that already existed. I was aiming for consistency more than clarity. There were a number of `.c`/`.out` files, but the majority was `.in`/`.out`.

This change also doesn't run any tests based on reading files. That's a future change :)